### PR TITLE
Hide the WP admin menu and toolbar on full screen pages

### PR DESCRIPTION
**Pull request by stephywells**
This should make scrolling eaiser

Fixes https://github.com/Strategy11/formidable-pro/issues/3918
(I think)

### CODE REVIEW

File: classes/controllers/FrmAppController.php [hunk 1]
```diff
@@ -461,6 +463,27 @@ public static function admin_init() {
...
+			return;
+		}
+
+		wp_deregister_script( 'admin-bar' );
+		wp_deregister_style( 'admin-bar' );
+		remove_action( 'admin_footer', 'wp_admin_bar_render', 1000 );
+		add_action(
+			'admin_head',
+			function() {
+				echo '<style>html.wp-toolbar{padding-top:0}</style>';
```
Comment (Crabcyborg) [inline classes/controllers/FrmAppController.php:482]: @stephywells I don't think this code is reached in the styler.

`disable_admin_menus` is only called `if ( FrmAppHelper::is_admin_page( 'formidable' ) ) {`. So this isn't applying everywhere full screen at the moment.

Screen Shot 2023-02-08 at 12 39 45 PM

**Comment by Crabcyborg** [comment]
It also looks like this update causes visual views to load as a white screen. I don't see any errors though.

Screen Shot 2023-02-08 at 12 44 00 PM

File: classes/controllers/FrmAppController.php (hunk 1 above)
Comment (stephywells) [reply classes/controllers/FrmAppController.php:482]: This is fixed now.

**Review by Crabcyborg — APPROVED** [review]
🚀

**Review by garretlaxton — APPROVED** [review]
This looks good for me.

**Pull request merged**